en/components/general/how-to/how-to-customize-theme.md

```diff
@@ -120,6 +120,94 @@ And the result in our app now looks like this:
 
 The same process can be applied to override and customize any of the component themes individually.
 
+### Switching custom themes at runtime
+
+Now let's dig even deeper and create two custom versions of the theme, which can be switched between at runtime. We can do this with user control/preference and let them switch it at any time. For the example, however, we will use the OS defined user preference (light or dark) in order to apply a theme, which matches the current OS setting. In order to do this, we will need two color palettes:
+
+```scss
+@use "minireset.css/minireset";
+@use "@infragistics/igniteui-angular/theming" as *;
+
+@include core();
+@include typography($font-family: "Poppins");
+
+$primary-dark: #1028c7;
+$primary-light: #3c55f1;
+$secondary-dark: #e0d94c;
+$secondary-light: #b4a904;
+
+$custom-palette-dark: palette(
+  $primary: $primary-dark,
+  $secondary: $secondary-dark,
+  $surface: #000,
+  $gray: #ccc,
+  $info: color($dark-material-palette, 'info'),
+  $success: color($dark-material-palette, 'success'),
+  $error: color($dark-material-palette, 'error'),
+  $warn: color($dark-material-palette, 'warn')
+);
+
+$custom-palette-light: palette(
+  $primary: $primary-light,
+  $secondary: $secondary-light,
+  $surface: #fff,
+  $gray: #222,
+  $info: color($light-material-palette, 'info'),
+  $success: color($light-material-palette, 'success'),
+  $error: color($light-material-palette, 'error'),
+  $warn: color($light-material-palette, 'warn')
```

Comment:
This is no longer needed, it's been patched in a [Ignite UI Theming v1.2.0](https://github.com/IgniteUI/igniteui-theming/releases/tag/1.2.0).